ts/src/htx.ts

```diff
@@ -941,16 +941,14 @@ export default class htx extends Exchange {
                 'fetchMarkets': {
                     'types': {
                         'spot': true,
-                        'future': {
-                            'linear': true,
-                            'inverse': true,
-                        },
-                        'swap': {
-                            'linear': true,
-                            'inverse': true,
+                        'linear': true,
+                        'inverse': {
+                            'future': true,
+                            'swap': true,
```

Comment:
we shouldn't do this inverts the hierarchy we have for markets type-> subtype

Reply:
but this is not for hierarchy, just a dict for fetching markets

Reply:
ok, i'll reorg that, will need few mins

Reply:
because that logic does not either fits in our `exceptions per market type` functionality, where, i.e. for binance, we have `spot, linear, inverse ...` flat hierarchy too, instead of `spot, swap, future ...`